### ISSUE TITLE
Interval problem on IOS

### ISSUE THREAD

**Issue by amanitequeen** [wbhob/ionic-long-press#13]
Hello, 

I have the same problem as the one dexscribe here : https://github.com/wbhob/ionic-long-press/issues/4 except i have my ionic-long-press at 1.5.0 version. The bug is fixed on Android but still observed in IOS.

**Comment by wbhob** [comment]
I don't know what this issue is. Can you share a repo or an implementation that I can test?

**Comment by amanitequeen** [comment]
Oh ok sorry

I have this code :
(html)
```
<button ion-item ion-long-press [interval]="500" (onPressing)="showstuff()" (click)="otherfunction()" *ngIf="true">
     <ion-icon name="clipboard">
</button>
```

(ts)
```
showstuff(): void {
        if(!isvisible){
            this.vibration.vibrate(100);
            this.hidestuff();
            isvisible = true;
        }
    }

hidestuff(): void{
       if(isvisible){
            isvisible = false;
        }
}

otherfunction(): void{
     console.log("clicked")
}
```
If I long press the button normally, everything works fine. But if I move my finger during the long press, the function "showstuff" loops and is called every 500ms. There was the same problem declared in issue #4 , and a fix was published in 1.3.0, but I observed the same problem only in IOS and my version of ionic-long-press is 1.5.0.

Thanks by advance

**Comment by wbhob** [comment]
This could be related to using click with this element. Does it work if you remove the click handler entirely?

**Comment by Adilkp796** [comment]
I solved this issue by add (tap) event instead of (click)..

**Comment by amanitequeen** [comment]
I already tried to remove the click event from the button, but the problem is still observed

**Comment by wbhob** [comment]
Have you tried @Adilkp796 's solution? This may just be a problem with mobile web/WebKit, in which case, I can't do anything to fix it.

**Comment by amanitequeen** [comment]
I tried with tap or without tap or click event

**Comment by wbhob** [comment]
Ok, but this could just be an issue with how the browser event engine works. This library is built around Hammer.JS, which takes event inputs directly from the browser. If the browser event doesn't fire correctly, as it seems to be the case, I can't do anything about it on my end.

From what you've told me, it seems like this is a browser issue because you're doing a *non-standard action*. Most users won't press on the button and move their finger away. When you do this action, the browser interprets it differently, and that is out of my control.

**Comment by amanitequeen** [comment]
Actually this bug happens very frequentely, because if you move finger just a bit, the bug happens. Sometimes it happens even when I carefully move finger. But if you say you can't do anything, I'll try someting else within the code to "delete" the loop.

**Comment by amanitequeen** [comment]
Solved it by updating IOS to IOS 12.0
thanks !
